src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SUPPLY_OPERATION = "supply";
+    private static final String BUY_OPERATION = "buy";
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        List<String> readedFile;
+        int supplyValue = 0;
+        int buyValue = 0;
+        int result = 0;
```

Comment:
The variable 'result' should be a local variable inside the try block where it's used since it's not required to be in the scope of the entire method.